compute/cker/src/train/BinaryArithmetic.test.cc

```diff
@@ -217,8 +217,78 @@ TEST(CKer_Operation, MulGrad)
   }
 }
 
-TEST(CKer_Operation, DistinctShape)
+TEST(CKer_Operation, neg_DistinctShape)
```

Comment:
Test name is now explicitly marked as negative.

Reply:
To avoid test name duplication, please add `BinaryArithmetic` prefix to the test name. :)
```suggestion
TEST(CKer_Operation, neg_BinaryArithmeticDistinctShape)
```

Reply:
@Aeren1564 In addition, please update commit title also. It would be better to specify `BinaryArithmetic` and `test`

Reply:
> To avoid test name duplication, please add BinaryArithmetic prefix to the test name. :)

Updated.

> @Aeren1564 In addition, please update commit title also. It would be better to specify BinaryArithmetic and test

Updated.